config/static_provider.go

```diff
@@ -20,13 +20,7 @@
 
 package config
 
-import (
-	"fmt"
-	"sync"
-)
-
 type staticProvider struct {
-	sync.RWMutex
```

Comment:
Well, we can get value directly:
https://github.com/uber-go/fx/blob/master/config/value.go#L277
So this lock doesn't do anything and we should copy map.